astropy/utils/console.py

```diff
@@ -498,6 +498,19 @@ def human_file_size(size):
     return "{0:>3s}{1}".format(str_value, suffix)
 
 
+class _mapfunc(object):
```

Comment:
The decorated function needs to be pickle-able, though. That wouldn't be pickle-able.